src/builtin.d

```diff
@@ -48,35 +49,35 @@ extern (C++) Expression eval_sin(Loc loc, FuncDeclaration fd, Expressions* argum
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, sinl(arg0.toReal()), arg0.type);
+    return new RealExp(loc, TargetReal.sin(arg0.toReal()), arg0.type);
```

Comment:
Simple:

* Decoupling both modules: with a custom `real_t` type, one only has to touch 1 module, leading to either
  * no redundant forwarders from `Port` to the actual implementations in `real_t.d`, or
  * no `real_t` specific implementations in `port.d` (excluding `yl2x` for a moment).
* Less imports in other modules.